src/plugins/intel_npu/src/plugin/npuw/util.cpp

```diff
@@ -886,23 +886,19 @@ void unpack_u4f16_asymm_zp(const ov::SoPtr<ov::ITensor>& from,
     const auto& from_shape = from->get_shape();
     NPUW_ASSERT(from_shape.back() % 64 == 0);
 
-    // 3-channel (group-wise) scale factors are
-    // supported.
-
+    // 2-channel (Symmetric) and 3-channel (group-wise)
```

Comment:
I'm not so sure about this change. What is asymm unpack? Is it always 3 channel parameters?

Reply:
you can discuss it with @ujjayant-kadian , he is a master of this question

Reply:
Already discussed, thank you!